test/errors/test_FPDF_errors.py

```diff
@@ -19,6 +17,15 @@ def test_add_page_throws_without_page():
     assert str(e.value) == msg
 
 
+def test_encoding_exception():
+    pdf = fpdf.FPDF()
+    pdf.add_page()
+    pdf.set_font("Helvetica", size=15)
+    with pytest.raises(FPDFException) as e:
```

Comment:
👍 for the unit test

You could check that a `FPDFUnicodeEncodingException` is raised here

Reply:
Changed to FPDFPUnicodeEncodingException